libwds/CMakeLists.txt

```diff
@@ -2,7 +2,26 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -fvisibility=hidden")
 set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} -std=c99 -Wall")
```

Comment:
It was there from the beginning. gcc defaults to 'GNU dialect of ISO C90', so setting to C99 is a better choice.